### ISSUE TITLE
Pod库 swift4.2报错

### ISSUE THREAD

**Issue by 824092805** [alexaubry/BulletinBoard#123]
BLTNInterfaceBuilder类中titleLabel.accessibilityTraits |= UIAccessibilityTraits.header报错Binary operator '|=' cannot be applied to two 'UIAccessibilityTraits' operands

**Comment by alexaubry** [comment]
Hi! Could you share this information so I can help:

- What dependency manager you use (CocoaPods / Carthage / manual)
- What version of the framework you use (2.0.2 is the only one that can be used with Xcode 10)

Thanks!

**Comment by yarodevuci** [comment]
I have error compiling on Swift 4.2 @alexaubry Please advise

**Comment by alombard** [comment]
Hi @alexaubry - first off great work - love the framework.  

I can confirm what @yarodevuci is seeing.

I'm using cocoapods on 2.0.2 and this is the only framework left of all that I use that seems to have this deprecation warning:

![image](https://user-images.githubusercontent.com/10245151/50087840-5f492d00-01cf-11e9-8fdb-03f9e5040d1b.png)

I've tried running this through the conversion tooling in Xcode but end up with some very strange errors - will try to take a look and post more detail here but suspect if you try to convert to Swift 4.2 syntax you will see the same.

Thanks in advance for your attention!

Andrew

**Comment by alexaubry** [comment]
This has been fixed in the latest release. You can all update the library to 3.0 in your Podfile for full support of Swift 4.2 😄

**Comment by alombard** [comment]
Awesome - Thanks!

________________________________
